Lib/multiprocessing/forkserver.py

```diff
@@ -93,6 +97,17 @@ def connect_to_new_process(self, fds):
                       resource_tracker.getfd()]
             allfds += fds
             try:
+                if self._forkserver_authkey:
```

Comment:
Am I right in understanding that authentication is disabled if `self._forkserver_authkey` is cleared?  In what situation would we expect this to not be set?  Should this be tested?

Reply:
Also, should there be a distinction between `None` and `b''` (or any other "false" value)?  IOW, should this be `if self._forkserver_authkey is not None`?

Reply:
good point, `ensure_running()` guarantees that `self._forkserver_authkey` is set.  conditional removed.